scripts/west_commands/cmake.py

```diff
@@ -39,6 +49,11 @@ def run_cmake(args, cwd=None, capture_output=False):
         kwargs['stderr'] = subprocess.STDOUT
     if cwd:
         kwargs['cwd'] = cwd
+
+    if dry_run:
+        log.inf('Dry run:', quote_sh_list(cmd))
```

Comment:
`log.inf` is always printed right?

Reply:
For now, yeah. We may add a -q switch later, but we don't have one yet.